test/repos/test_zoo_yml.py

```diff
@@ -65,10 +76,17 @@ def test_generate(generate_services_with_environments):
   - https://serviceurl2
 - name: staging
   dashboard_url: https://dashboardurl
-  health_check_url: https://healthcheckurl
+  health_check_url: null
   service_urls:
   - https://serviceurl1
   - https://serviceurl2
+arbitrary_links:
+- name: Datadog
+  url: https://dashboard.datadog.com
+  emoji: ':simple_smile:'
```

Comment:
Either we have to live with fact that emoji will be dumped with `''` around it or we have to store it without `::`

Reply:
hmm... how are we rendering the emojis on the frontend? 🤔 
I don't see any frontend change to display this data 😄️ 
so... I'd defer this decision until we decide how we'll display them